xmtp_api_grpc/src/grpc_api_helper.rs

```diff
@@ -57,8 +63,18 @@ pub enum InnerApiClient {
     ),
 }
 
+pub enum InnerMlsClient {
```

Comment:
Confirming I understood here - the MlsClient is NOT the validation service, it is a client for the golang gRPC server?

Reply:
That's right.

`xmtp_mls` talks to `xmtp-node-go` which talks to `mls_validation_service`